src/estimagic/optimization/subsolvers/bounded_newton_quadratic.py

```diff
@@ -216,14 +251,31 @@ def compute_conjugate_gradient_step(
                     options_update_radius["max_radius"],
                 )
 
-                step_inactive = minimize_trust_conjugate_gradient(
-                    gradient_inactive,
-                    hessian_inactive,
-                    trustregion_radius,
-                    gtol_abs=gtol_abs_conjugate_gradient,
-                    gtol_rel=gtol_rel_conjugate_gradient,
-                )
-                step_norm = np.linalg.norm(step_inactive)
+                if conjugate_gradient_method == "standard":
+                    step_inactive = minimize_trust_cg(
+                        gradient_inactive,
+                        hessian_inactive,
+                        trustregion_radius,
+                        gtol_abs=gtol_abs_conjugate_gradient,
+                        gtol_rel=gtol_rel_conjugate_gradient,
+                    )
+                    step_norm = np.linalg.norm(step_inactive)
+                elif conjugate_gradient_method == "steihaug-toint":
```

Comment:
Here it's still `steihaug-toint`, above it was `steihaug_toint`. Make sure it's consistent everywhere and I prefer the `_` version.